src/handler/fileOperations.ts

```diff
@@ -366,6 +366,52 @@ export class FsspecModel {
     }
   }
 
+  async sync_push(
```

Comment:
We may wish to eventually provide a feedback mechanism (see e.g., https://filesystem-spec.readthedocs.io/en/latest/features.html#callbacks ).